flink-end-to-end-tests/test-scripts/test-runner-common.sh

```diff
@@ -105,6 +107,13 @@ function post_test_validation {
     fi
 }
 
+function log_environment_info {
+    echo "Jps"
+    jps
```

Comment:
If you are going with `jps`, I would use `jps -v`, as it also lists the arguments (which sometimes include the log directory, which might help identifiying where the process is coming from).

Secondly, I would also run `pstree`. Some tests (k8s, dockerized ...) are running non-JVM processes.

Reply:
+1 for -v, +-0 for pstree since adds a fair amount of noise that I'm right now not really interested in. You'd rather want something that creates a diff of processes. As it stands it would not be obvious which remaining processes are interesting/problematic.

Reply:
Yes, pstree would be a lot of output. You could consider logging it to a file. But that's up to you